{{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/urls.py

```diff
@@ -12,6 +11,7 @@
 ]
 
 if settings.DEBUG_TOOLBAR:
+    import debug_toolbar
     urlpatterns += [
         path('__debug__/', include(debug_toolbar.urls)),
```

Comment:
```suggestion
        path('__debug__/', include("debug_toolbar.urls")),
```

You should be able to change the `include` to use a string instead, then you don't need to import the debug toolbar at all.

Reply:
If the string passed to `include()` represents a path that cannot be imported (because it's spelled incorrectly, for example), will the exception trigger eagerly during application start or lazily, during the first call that matches the url pattern?

Reply:
I believe Django imports all URLconf entries when starting up.

Reply:
@agoncharov-reef  Sure.